lib/linter/linter.js

```diff
@@ -2043,9 +2150,12 @@ class Linter {
             fixedResult,
             fixed = false,
             passNumber = 0,
+            fixPasses = 0,
             currentText = text;
         const debugTextDescription = options && options.filename || `${text.slice(0, 10)}...`;
         const shouldFix = options && typeof options.fix !== "undefined" ? options.fix : true;
+        const stats = options && typeof options.stats !== "undefined" ? options.stats : false;
```

Comment:
I think we can simplify a bit. Based on usage, I think it's fine to leave `stats` as `undefined` rather than setting to `false`, but let me know if I'm missing something.
```suggestion
        const stats = options?.stats;
```